recipes/pdf_recipe/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "diffpy.pdfmorph" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diffpy/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 699952c747048db4a383e81fa867887a86480200a56d7a51bdd1ad43d25b392d
+
+build:
+  skip: True  # [py<3]
```

Can this be `noarch`?
```suggestion
  noarch: python
```

---

It can't, it ships a `.exe` file.

---

It looks like it's an `entry_point`? If so, [this](https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points) should be included in the metadata

---

I can confirm that `conda install` does create an `.exe` launcher for a `noarch` package with `entry_point` defined.